### PR TITLE
Update Locate bark.ipynb

### PR DESCRIPTION
**Pull request by DanteEnjoyer**
Quick fix for issue in script

### CODE REVIEW

File: 3d aproksymacja/Locate bark.ipynb [hunk 1]
```diff
@@ -79,13 +79,12 @@
...
-    "n = points.shape[0]\n",
-    "nu = points1.shape[0]\n",
-    "nub = points2.shape[0]\n",
     "points = np.array(points).T\n",
     "points1 = np.array(points1).T\n",
     "points2 = np.array(points2).T\n",
-    "print(points.shape)\n",
+    "n = points.shape[1]\n",
+    "nu = points1.shape[1]\n",
+    "nub = points2.shape[1]\n",
```
Comment (MateuszAdamski7) [inline 3d aproksymacja/Locate bark.ipynb]: what are those variable names

Comment (jan-stasiak) [reply 3d aproksymacja/Locate bark.ipynb]: Could you please remove the comments in Polish, thank you from the mountain for resolving the issue quickly

Comment (jan-stasiak) [reply 3d aproksymacja/Locate bark.ipynb]: Could you please not use magic numbers. For an overview, I provide the following link:

https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=In%20computer%20programming%2C%20a%20magic,see%20List%20of%20file%20signatures

**Review by jan-stasiak — CHANGES_REQUESTED** [review]

**Comment by jan-stasiak** [comment]
Whilst I generally find myself harboring a certain measure of disapproval towards the intertwining of variable nomenclature and commit designations, I shall, on this occasion, permit myself to indulge in a modicum of leniency and overlook this particular discrepancy. We shall allow this nonconformance to pass unchecked, albeit for today.

**Comment by jan-stasiak** [comment]
Whilst I may exhibit leniency today and tolerate these oversights, it remains imperative, nay, an obligation, that you undertake a focused and unswerving endeavour to eradicate these transgressions. With the sands of time serving as your pedagogue, I implore you to embrace refinement, mastery and the uncompromising pursuit of perfection. Allow not these mishaps to besmirch your noble journey; instead let them be stepping stones in your magnificent ascension towards an unparalleled excellence.

**Review by jan-stasiak — DISMISSED** [review]
While it is not entirely to my taste, I find it necessary to acquiesce gracefully.

**Review by Barstok — CHANGES_REQUESTED** [review]
אני לא מסכים

**Review by Barstok — APPROVED** [review]
מגניב

**Pull request merged**